src/scss/patterns/_gallery.scss

```diff
@@ -39,6 +39,7 @@ figure {
   }
 }
 
+figcaption,
```

Comment:
@mirisuzanne I was going to add a class of caption to the figcaptions in the md file and the work page, but I saw many figcaptions that didn't have the .caption class, but they probably could benefit from the styles. Is there a reason any figcaption shouldn't be slightly smaller to stand out?

Reply:
nope - there's probably just legacy markdown that we ported from rstBlog and never cleaned up.